web/src/render.tsx

```diff
@@ -17,11 +17,56 @@
 import './main.scss';
 import ReactDom from 'react-dom';
 import React from 'react';
+import { encrypt_string } from '../pkg';
 
 const hljs = require('highlight.js');
 (window as any).hljs = hljs;
 require('highlightjs-line-numbers.js');
 
+class PasteForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: "Sample text"
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event) {
+    try {
+      encrypt_string(this.state.value);
+    } catch (e) {
+      console.error(e);
```

Comment:
Can we give context to the error message pls?

Reply:
just removed the try/catch, error printing has been punted to rust